admin_manual/enterprise_user_management/user_auth_shibboleth.rst

```diff
@@ -129,7 +157,8 @@ Application Configuration
 
 After installing and enabling the Shibboleth application there are three 
 configuration variables to set up, depending on the data sent back by the 
-IdP. The configuration is stored in ``config.php``:
+IdP. The configuration is stored in ``config/config.php``, or you may create a 
```

Comment:
Unfortunately, the app uses its own configuration in `apps/user_shibboleth/config.php` @DeepDiver1975 should we move that to the core config.php ar make it configurable in the admin ui and store it in the db? I assume the latter. Tracked in https://github.com/owncloud/user_shibboleth/issues/54